tests/framework/framework.go

```diff
@@ -76,34 +80,42 @@ func init() {
 // NewFramework makes a new framework and sets up the global BeforeEach/AfterEach's.
 // Test run-time flags are parsed and added to the Framework struct.
 func NewFramework(prefix string) *Framework {
+	var err error
```

Comment:
defining err here is very C-like. The go approach is, when possible, define your local vars directly in an assignment. When that won't work (say due to scoping rules) then define the var right before it's needed. In this case you do not need a definition for `err`

Reply:
Gotcha, well I can assign with := so removing.